src/Logger.php

```diff
@@ -91,7 +91,7 @@ protected function __construct(string $minLevel = LogLevel::WARNING, ?\Closure $
             throw new InvalidArgumentException(\sprintf('The log level "%s" does not exist.', $minLevel));
         }
 
-        $this->minLevelIndex = self::$levels[$minLevel];
+        $this->minLevelIndex = \min(self::$levels[$minLevel], self::$levels[self::$madelineLevels[MadelineProto\Logger::VERBOSE]]);
```

Comment:
Nono :) I have slow network disk, cant afford this many logs.

Reply:
Then completely disable logs if the loglevel is below verbose

Reply:
I need logs non verbose logs. And i need ability to change level :)